go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/sourcenetwork/defradb
 
-go 1.22.0
+go 1.22.7
 
-toolchain go1.22.7
+toolchain go1.22.8
```

Comment:
@sourcenetwork/database-team  cometbft@v0.38.15 strictly requires go >= 1.22.7 so I had to run `make tidy`. Are we okay with this change?